packages/sanity/src/core/studio/components/navbar/resources/ResourcesButton.tsx

```diff
@@ -0,0 +1,46 @@
+import {HelpCircleIcon} from '@sanity/icons'
+import {Box, Button, Flex, Menu, MenuButton, Text, Tooltip} from '@sanity/ui'
+import React from 'react'
+import styled from 'styled-components'
+import {useColorScheme} from '../../../colorScheme'
+import {useGetHelpResources} from './helper-functions/hooks'
+import {ResourcesMenuItems} from './ResourcesMenuItems'
+
+const StyledMenu = styled(Menu)`
+  max-width: 300px;
+  min-width: 200px;
+`
+
+export function ResourcesButton() {
+  const {scheme} = useColorScheme()
+
+  const {value, error, isLoading} = useGetHelpResources()
+
+  return (
+    <Flex>
+      <Tooltip
+        content={
+          <Box padding={2}>
+            <Text size={1}>Help and Resources </Text>
```

Comment:
Is the Title Case intentional here? It sounds kinda like a product, but... it isn't? :)
Not sure if this is a discussed and decided point - feel free to ignore if so.

```suggestion
            <Text size={1}>Help and resources</Text>
```